numba/targets/arrayobj.py

```diff
@@ -1706,6 +1706,60 @@ def scalar_sinc_impl(val):
                                    locals=dict(c=scalar_dtype))
     return impl_ret_untracked(context, builder, sig.return_type, res)
 
+@builtin
+@implement(numpy.angle, types.Kind(types.Number))
```

Comment:
I think this should all be in `numba.targets.npyfuncs`, with other similar functions.